containers/runner.Containerfile

```diff
@@ -153,13 +154,9 @@ RUN --mount=type=cache,target=/var/cache/pacman/pkg \
     paccache -rk1 && \
     rm -rf /usr/share/{info,man,doc}/*
 
-### Application Installation ###
-ARG LUDUSAVI_VERSION="0.28.0"
-RUN curl -fsSL -o ludusavi.tar.gz \
-        "https://github.com/mtkennerly/ludusavi/releases/download/v${LUDUSAVI_VERSION}/ludusavi-v${LUDUSAVI_VERSION}-linux.tar.gz" && \
-    tar -xzvf ludusavi.tar.gz && \
-    mv ludusavi /usr/bin/ && \
-    rm ludusavi.tar.gz
+### SSH Configuration ###
+RUN echo "PermitRootLogin no" >> /etc/ssh/sshd_config && \
+    echo "PasswordAuthentication yes" >> /etc/ssh/sshd_config
 
```

Comment:
I don't like the idea to have standard passwords and password authentication. Please we need to find a better way for ssh access!
Don't merge that.

Reply:
It's not enabled by default, and it's meant for debugging only.

Reply:
I've added public-key only authentication now, sorry for the delay!
Please resolve and review if you deem the changes good and safe :pray:

Reply:
Resolved as per Discord messages.